### DISCUSSION TITLE
【重要】关于项目新名称的投票

### DISCUSSION THREAD

**Discussion by Xiaokang2022** [Xiaokang2022/maliang#41] [Polls]
目前该项目的名称已经不再适合当前的状况。当前项目的能力已经远超一个工具的能力上限，定性为微型框架比较合适。

**那到底有没有必要在下个正式版本（`3.0.0`）之前换一个新的名字呢？**

一些考虑因素：

* 不换，会对其他已使用该项目造成一定的影响；
* 不换，会丢失原有项目的 PyPI 数据（需要新建一个 PyPI 项目）；
* 不换，新的名称可能不容易宣传；
* 换，名称与项目不符；
* 换，新的名称可能更利于项目宣传；
* 换，原名称容易误导新用户，他们会认为这是个小工具，而非全新的框架；

在下面给出你们的投票吧！（选择有必要的朋友，可以在下面评论区给出你的希望的新名称供大家参考！）

投票截止日期（可能更改）：2025/01/01

> [!TIP]
> 不了解该项目？您可以通过该项目的[官网](https://xiaokang2022.github.io/tkintertools/)、[3.0 预览版本文档网站](https://xiaokang2022.github.io/tkintertools-docs/3.0/) 来了解本项目。此外，还可以通过一些介绍该项目的[视频](https://space.bilibili.com/498105668/channel/collectiondetail?sid=2710548&ctype=0)了解该项目。

**Comment by wtwang1998** [comment]
长痛不如短痛，建议丢掉历史包袱，同时在文档中单独给出迁移指南。

**Reply by Xiaokang2022** [reply]
大佬言之有理啊，看来起一个新名字的压力要给到了我身上了😂

**Comment by Xiaokang2022** [comment]
感谢各位的投票！根据投票结果，该项目在 `3.0.0` 版本发布时会更换一个新的名称。具体名称在发布新版本时会进行说明，感谢大家的支持！
